<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSequenceDialog</class>
 <widget class="QDialog" name="NewSequenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>466</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Cabin</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>New Sequence</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="mainHeader">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cabin</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New Sequence</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="mainFrame">
     <property name="font">
      <font>
       <family>Cabin</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(136, 136, 136);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QLabel" name="notesLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Notes:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameValue">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTextEdit" name="sequenceValue">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>561</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cousine</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="PrimerDesign::ValidatorLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="notesValue">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>561</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="PrimerDesign::ValidatorLabel" name="sequenceLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sequence:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="dialogButtonFrame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QDialogButtonBox" name="dialogChoices">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QDialogButtonBox {
	padding: 10px;
}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PrimerDesign::ValidatorLabel</class>
   <extends>QLabel</extends>
   <header>forms/PrimerDesign/Labels.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameValue</tabstop>
  <tabstop>sequenceValue</tabstop>
  <tabstop>notesValue</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dialogChoices</sender>
   <signal>accepted()</signal>
   <receiver>NewSequenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogChoices</sender>
   <signal>rejected()</signal>
   <receiver>NewSequenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sequenceValue</sender>
   <signal>textChanged()</signal>
   <receiver>NewSequenceDialog</receiver>
   <slot>enableOkIfValidForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameValue</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewSequenceDialog</receiver>
   <slot>enableOkIfValidForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameValue</sender>
   <signal>lostFocus()</signal>
   <receiver>NewSequenceDialog</receiver>
   <slot>enableOkIfValidForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableOkIfValidForm()</slot>
 </slots>
</ui>
